components/automate-ui/src/app/entities/service-groups/service-groups.effects.ts

```diff
@@ -64,9 +64,9 @@ export class ServiceGroupsEffects {
   @Effect()
   getServiceGroupsCounts$ = this.actions$.pipe(
       ofType(ServiceGroupsActionTypes.GET_SERVICE_GROUPS_COUNTS),
-      withLatestFrom(this.store),
-      switchMap(([_action]) => {
-        return this.requests.fetchServiceGroupHealth().pipe(
+      withLatestFrom(this.store.select(serviceGroupsFilters)),
+      switchMap(([_action, filters]: [any, ServiceGroupsFilters]) => {
+        return this.requests.fetchServiceGroupHealth(filters).pipe(
         map((payload: ServiceGroupsHealthSummary) => new GetServiceGroupsCountsSuccess(payload)),
```

Comment:
[nit] Is indentation correct here? I'd expect a level of nesting more. 🤔

Reply:
@srenatus Updated the indentation :)